ve/cuda/engine_cuda.cpp

```diff
@@ -111,7 +111,7 @@ EngineCUDA::~EngineCUDA() {
 
     // File clean up
     if (not verbose) {
-        fs::remove_all(tmp_src_dir);
+        fs::remove_all(tmp_src_dir);texec
```

Comment:
What is is?

Reply:
No idea, sorry. I couldn't test the CUDA implementation

Reply:
Actually, the CUDA implementation is unfinished - seems that I forgot the final touches. I will update.

Reply:
:+1: